src/core/include/openvino/opsets/opset.hpp

```diff
@@ -10,7 +10,18 @@
 #include <set>
 #include <utility>
 
+#ifndef IN_OV_COMPONENT
+#    define IN_OV_COMPONENT
+#    define WAS_OV_LIBRARY_DEFINED_SERIALIZE
+#endif
+
 #include "ngraph/factory.hpp"
```

Comment:
This is probably conflict with https://github.com/openvinotoolkit/openvino/pull/22371

Reply:
Will sync depends which one merge first. I think factory removal will be first so then remove this include.
Without it build will fail.